Core/scripts/dirac-install.py

```diff
@@ -843,11 +843,17 @@ def downloadAndExtractTarball( tarsURL, pkgName, pkgVer, checkHash = True, cache
   tarName = "%s-%s.tar.gz" % ( pkgName, pkgVer )
   tarPath = os.path.join( cliParams.targetPath, tarName )
   tarFileURL = "%s/%s" % ( tarsURL, tarName )
+  tarFileCVMFS = "/cvmfs/dirac.egi.eu/installSource/%s" % tarName
```

Comment:
Is this cvmfs repository installed on _all_ grid sites?
Would it be possible to make this an option so we could point to our own cvmfs repositories?